instrumentation/opentelemetry-instrumentation-dbapi/src/opentelemetry/instrumentation/dbapi/__init__.py

```diff
@@ -275,6 +280,60 @@ def __init__(
         self.name = ""
         self.database = ""
         self.connect_module = connect_module
+        self.commenter_data = self.calculate_commenter_data()
+
+    def calculate_commenter_data(
+        self,
+    ):
+        commenter_data = {}
+        if not self.enable_commenter:
+            return commenter_data
+
+        try:
+            db_driver = self.connect_module.__name__
+        except AttributeError:
+            db_driver = "unknown"
```

Comment:
Maybe this instead?
```suggestion
        db_driver = getattr(self.connect_module, '__name__', 'unknown')
```

Reply:
Good idea! Changed in [c461544](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2897/commits/c461544d226437e552a1c6ffdabf8bf406fb9f38)